web-ng/src/app/services/navigation/navigation.service.ts

```diff
@@ -220,6 +221,13 @@ export class NavigationService {
     ]);
   }
 
+  /**
+   * Navigate to the URL for viewing the projects.
```

Comment:
Nit: => "for viewing a list of available projects"

Reply:
updated.